Sources/Remote Logging/Crash Logging/CrashLogging.swift

```diff
@@ -56,6 +56,10 @@ public class CrashLogging {
             options.diagnosticLevel = .error
 
             options.environment = self.dataProvider.buildType
+            switch self.dataProvider.releaseName {
+            case .setByApplication(let name): options.releaseName = name
+            case .setByTracksLibrary: options.releaseName = nil
```

Comment:
Leaving a breadcrumb: this is bug (addressed in Android in: https://github.com/Automattic/Automattic-Tracks-Android/pull/209)